core/update.php

```diff
@@ -89,6 +90,7 @@ function update_script_selection_form($form, &$form_state) {
         '#suffix' => '</div>',
       );
       $incompatible_updates_exist = TRUE;
+      $incompatible_updates_messages[] = $update;
```

Comment:
Should we increment `$incompatible_count` here too ? I'm a little bit lost with the whole logic.